jina/executors/indexers/keyvalue.py

```diff
@@ -29,7 +29,7 @@ def flush(self):
     class ReadHandler:
         def __init__(self, path):
             with open(path + '.head', 'rb') as fp:
-                tmp = np.frombuffer(fp.read(), dtype=np.int64).reshape([-1, 4])
+                tmp = np.frombuffer(fp.read(), dtype=[np.object, np.int64, np.int64, np.int64]).reshape([-1, 4])
```

Comment:
I am pretty sure, that replacement must also happen in line 65 and 101 of this file.